Header header
int64 proximity_value
